src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
```

Comment:
@Yaoxuannn you forgot to commit reportWebVitals, didn't you?

Reply:
Right, this should be remove. Thanks!